djangocms_navigation/utils.py

```diff
@@ -37,4 +37,4 @@ def get_versionable_for_content(content):
     try:
         return versionables.for_content(content)
     except KeyError:
-        pass
+        return
```

Comment:
What's the difference?

Reply:
IMO, `pass` is to create an empty block where the return is to let control flow to continue which is more readable I think ;)